enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/load_balancing/procedure/GetServersProcedureV1.java

```diff
@@ -117,29 +111,28 @@ public ProcedureSignature signature()
         }
 
         return discoveryService.coreServers().find( leader )
-                .map( server -> server.getClientConnectorAddresses().getBoltAddress() );
+                .map( server -> server.connectors().boltAddress() );
```

Comment:
Maybe use `Util#extractBoltAddress()` here and in all other places in this class that do `connectors().boltAddress()`?